crypto/bio/bss_file.c

```diff
@@ -329,15 +330,22 @@ static long file_ctrl(BIO *b, int cmd, long num, void *ptr)
             ret = 0;
         }
         break;
+    case BIO_CTRL_DGRAM_SET_NEXT_TIMEOUT:
```

Comment:
???  Why should the FILE BIO support a control that is unique to the datagram BIO

Reply:
See above.